.rubocop_todo.yml

```diff
@@ -238,6 +241,20 @@ RSpec/ReceiveNever:
   Exclude:
     - 'spec/beaker/hypervisor/vagrant_spec.rb'
 
+# Offense count: 7
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, IgnoreMetadata.
+# Include: **/*_spec.rb
+RSpec/SpecFilePathFormat:
+  Exclude:
+    - '**/spec/routing/**/*'
+    - 'spec/beaker/hypervisor/vagrant_custom_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_desktop_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_fusion_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_libvirt_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_parallels_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_virtualbox_spec.rb'
+    - 'spec/beaker/hypervisor/vagrant_workstation_spec.rb'
```

Comment:
Should we fix this by renaming `spec/beaker/hypervisor/*` to `spec/beaker/*`?

Reply:
moved it